src/odemis/gui/cont/streams.py

```diff
@@ -2534,6 +2533,39 @@ def _updateMicroscopeStates(self):
                 self._tab_data_model.emState.value = guimodel.STATE_OFF
 
 
+class CryoStreamsController(StreamBarController):
+    """
+    Controls the display of stream panels without affecting the actual streams
+    """
+
+    def _onView(self, view):
+        """ Handle the changing of the focused view """
+        if not view or self.ignore_view:
+            return
+
+        # hide/show the stream panels which are compatible with the view
+        allowed_classes = view.stream_classes
+        for e in self._stream_bar.stream_panels:
+            e.Show(isinstance(e.stream, allowed_classes))
+            ov_streams = self._tab_data_model.overviewStreams
+            static = issubclass(view.stream_classes, StaticStream)
+            # Check view type between Acquired and Overview
+            if static and ov_streams.value:
```

Comment:
Why do you have to check for `ov_streams`? It cannot be None. It's just an empty list. So I don't think it's required. If it is, I think it's worth a comment.